flume-ng-core/src/test/java/org/apache/flume/source/shaded/guava/TestRateLimiter.java

```diff
@@ -0,0 +1,512 @@
+package org.apache.flume.source.shaded.guava;
```

Comment:
some tests are commented out, if they are not needed, I believe they could be removed

Reply:
Fixed